bare/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+)
+
+type buildbarnProcess struct {
+	config string
+	binary string
+}
+
+func bbStart(bbProcess buildbarnProcess) (*os.Process, error) {
```

Comment:
This is just a tiny program. No need to do any actual error propagation. It's all right to just call `log.Fatal()`, right?